docs/examples/writing_drivers/Creating-Instrument-Drivers.ipynb

```diff
@@ -83,8 +112,34 @@
     "  \n",
     "Golden rule: if a `Parameter` is settable, it must always accept its own output as input.\n",
     "\n",
-    "In most cases you will probably be adding parameters via the `add_parameter` method on the instrument class as shown in the example below. \n",
+    "### Adding parameters to intruments"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
```

Comment:
a question about these raw cells - i find it not so cool that github's notebook rendering ignores them (see here https://github.com/QCoDeS/Qcodes/blob/ad2ea22cdeeea7be7432b2d8876e84e7f72abcb1/docs/examples/writing_drivers/Creating-Instrument-Drivers.ipynb). would you agree to split the content of these cells into parts that can be simple markdown and parts the require the rst syntax? :) this will have additional benefit (not so big but is nice and welcoming) that the non-rst content can be edited/improved by contributors who are less comfortable with rst but are comfortable with markdown. what do you think? :)

Reply:
Good catch, made the changes to avoid using rst as much as possible. I used it because it is so much easier to link to different parts of the docs